package.json

```diff
@@ -4,9 +4,13 @@
   "workspaces": [
     "packages/*"
   ],
+  "scripts": {
+    "build": "tsc --build tsconfig.json && cd ./packages/explorer && yarn build"
```

Comment:
Why do we need to build explorer separated?

Reply:
The 'explorer' is made with reactjs so we need to run `react-scripts build` separated. React script will handle the tsconfig file